pallets/stellar-relay/src/tests.rs

```diff
@@ -569,8 +577,11 @@ fn update_tier_1_validator_store_old_organization_and_validator_and_block_height
 			public_key: Default::default(),
 			organization_id: organization.id,
 		};
-		let new_validator_set = vec![validator; 2];
-		let new_organization_set = vec![organization; 2];
+		let mut validator_2 = validator.clone();
+		validator_2.public_key = BoundedVec::<u8, FieldLength>::try_from(vec![1u8; 128]).unwrap();
```

Comment:
Here you are assigning the same key as in line 547. Is this a mistake or supposed to be like this? I would actually expect this test to fail now. Or with the changes you added in this PR we do want this test to fail. Because now we call the update extrinsic with two validators that have the same public key which is problematic and should throw the 'Duplicate...' error

Reply:
Validator and validator 1 has a different pub key.
test should fail and it works as expected.

Updated the test to specify more explicit public key with using 1 and 2 digit to fill the vector.